source/installation/proxy/iis.html.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Configuring IIS
+---
+
+If you want to use Plume for a **subdomain**, follow these instructions.
+
+1. You need to have installed URL Rewrite
+2. From the Sites, choose your site, in the middle pane, double click URL Rewrite
+3. On right hand side, Add Rule, Blank Rule
+4. Name it Plume with pattern (.*) with Action type Rewrite and Rewrite URL of http://127.0.0.1:7878/{R:1}
+5. Apply
+
+If you want to use Plume for a **subdirectory**, follow these instructions (experimental).
```

Comment:
I don't know if you tested this option, but I'm not sure it will actually work, since Plume was designed with the assumption that it was running on its own domain.

Reply:
Alright. That is why I put experimental as it is very difficult to implement, and probably wouldn't work anyway. I'll just remove it for now.

Reply:
we could open it as feature request.
it's probably somewhat related (orthogonal) to https://github.com/Plume-org/Plume/issues/94

Reply:
I’ve thought about this to. I think if you use a subdirectory variable in .env that accepts null and folder names. This sets a variable to all css and js (to be added) and adds correct path to css and js from web on server start. It is similar to adding a meta tag to the html something like `<meta name=“suburl” content=“/plume” />`

This is how Gitea does it... not that it means anything.